### PR TITLE
Add tests for speech modules

### PR DESCRIPTION
**Pull request by jakubl7545**
This PR adds unit tests for speech module. It translates different math content from LaTEX source code to more usable speech version. The content includes superscripts, roots, fractions and many others

### CODE REVIEW

File: latex_access/tests/test_speech.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        self.assertEqual(self.speech.sqrt('[3]{27}', 0), ('cube root 27', 7))
+        self.assertEqual(self.speech.sqrt('[5]{32}', 0), ('5th root 32', 7))
+
+    @unittest.skipIf(incompatible, "This test fails in Python 3 because of StringType attribute not available in Types module.")
+    def test_complex_roots(self):
+        """Tests translation of complex roots. Separated from other tests because it fails on Python 3."""
+        self.assertEqual(self.speech.sqrt('{2^2}', 0), (' begin  root 2 squared  end root', 5))
+
+    def test_frac(self):
+        """Tests translactions of fractions."""
```
Comment (lukaszgo1) [inline latex_access/tests/test_speech.py:52]: nit: translactions -> translations
```suggestion
        """Tests translations of fractions."""
```

**Review by lukaszgo1 — APPROVED** [review]
In general this looks pretty nice. I've executed these tests under the coverage, and you've managed to cover 95% of the speech module which is pretty impressive. If you want you  can try to add additional tests to bring the coverage to 100
% but I won't block my approval on it.

**Comment by jakubl7545** [comment]
I've added tests for more speech files but few things are untested. @lukaszgo1 Maybe you could look at these - especially translations of roots in both modified and hungarian speech and norma in hungarian. These two contain regular expressions which I can't figure out. The rest is not so important. Also do you have an idea how to fix failures on Python 3?

**Comment by lukaszgo1** [comment]
I've made these tests not fail Under Python 3, and added tests for roots in speech modified. Please review when you have time. As to the tests for Hungarian speech I'm not sure if spending any more time on these is a good idea - we don't speak the language so this is more of a guess work than real testing.

**Review by jakubl7545 — COMMENTED** [review]
@lukaszgo1 Looks good. I've added few more tests for hungarian speech which are similar to those you've added for speech_modified. Coverage for hungarian is now above 90%. I won't be adding any more tests here so unless you have some comments feel free to merge.

**Review by lukaszgo1 — APPROVED** [review]

**Comment by lukaszgo1** [comment]
According to Coverage some branches are not tested in both `speech` and `speech_modified`, however I was unable to quickly figure out how to cover them, so let's leave this for another day. I'm going to merge this PR.

**Pull request merged**